rpa-bot/1.1/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -60,3 +60,12 @@ The operating system can block the execution of certain files. Ensure that all e
 |RPA Builder | 6.1.1 and later
 |RPA Recorder | 6.0.0 and later
 |===
+
+== Internet Route Requirements
+
+To communicate with RPA Manager, RPA Bot requires you to enter one of the following Internet routes in the *RPA Manager to connect* field on the RPA Bot Configurator pop-up:
+
+* https://[your subdomain].rpa.mulesoft.com
+* https://*.cloudfront.net
```

Comment:
```suggestion
* +https://[your subdomain].rpa.mulesoft.com+
* +https://*.cloudfront.net+
```
This change prevents the URLs from being clickable, to avoid unwanted navigation.